src/openassetio-core/include/openassetio/managerApi/ManagerInterface.hpp

```diff
@@ -579,7 +579,7 @@ class OPENASSETIO_CORE_EXPORT ManagerInterface {
   /**
    * Callback signature used for a successful entity resolution.
    */
-  using ResolveSuccessCallback = std::function<void(std::size_t, const TraitsDataConstPtr&)>;
+  using ResolveSuccessCallback = std::function<void(std::size_t, const TraitsDataPtr&)>;
```

Comment:
[consider] Be good to add a release notes entry

Reply:
Added in 39948d2